app/src/test/java/com/example/benchproject/app/popular/movies/viewModel/PopularMoviesViewModelTest.kt

```diff
@@ -0,0 +1,82 @@
+package com.example.benchproject.app.popular.movies.viewModel
+
+import com.example.benchproject.app.popular.movies.entity.Movie
+import com.example.benchproject.app.popular.movies.view.PopularMoviesFragmentNavigator
+import com.example.benchproject.domain.popular.movies.entity.MovieModel
+import com.example.benchproject.domain.popular.movies.usecase.GetPopularMoviesUseCase
+import com.example.benchproject.test.common.InstantExecutorExtension
+import io.mockk.Runs
+import io.mockk.coEvery
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.amshove.kluent.shouldBe
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.extension.ExtendWith
+
+@ExtendWith(InstantExecutorExtension::class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+internal class PopularMoviesViewModelTest {
+
+    private val movieListModel = Result.success(
+        listOf(
+            MovieModel(10001, "Test1", 5.4, "2022-06-23", "https://image.tmdb.org/t/p/w500/something1.com"),
+            MovieModel(10002, "Test2", 6.4, "2022-06-22", "https://image.tmdb.org/t/p/w500/something2.com"),
+            MovieModel(10003, "Test3", 3.4, "2022-06-21", "https://image.tmdb.org/t/p/w500/something3.com")
+        )
+    )
+    private val movieList = listOf(
+        Movie(10001, "Test1", 5.4, "2022-06-23", "https://image.tmdb.org/t/p/w500/something1.com"),
+        Movie(10002, "Test2", 6.4, "2022-06-22", "https://image.tmdb.org/t/p/w500/something2.com"),
+        Movie(10003, "Test3", 3.4, "2022-06-21", "https://image.tmdb.org/t/p/w500/something3.com")
+    )
+    private val badResponseFromRepository =
+        Result.failure<List<MovieModel>>(
+            Throwable("Unable to resolve host \"api.themoviedb.org\": No address associated with hostname")
+        )
+    private val getPopularMoviesUseCase: GetPopularMoviesUseCase = mockk()
+    private val popularMoviesFragmentNavigator: PopularMoviesFragmentNavigator = mockk()
+
+    @BeforeAll
+    fun initAll() {
+        Dispatchers.setMain(Dispatchers.Unconfined)
+    }
+
+    @AfterAll
+    fun tearDown() {
+        Dispatchers.resetMain()
```

Comment:
Add it to base class `InstantExecutorExtension`

Reply:
I added it to this class